pkg/provider/azure.go

```diff
@@ -48,7 +48,6 @@ import (
 	cloudnodeutil "k8s.io/cloud-provider/node/helpers"
 	nodeutil "k8s.io/component-helpers/node/util"
 	"k8s.io/klog/v2"
-	"sigs.k8s.io/yaml"
 
 	"sigs.k8s.io/cloud-provider-azure/pkg/azclient"
```

Comment:
Looks like there is an issue with goimports checker

Reply:
If this pkg is not moved, will there be a failure?

Reply:
I don't think it should be here if not used.

Reply:
fix: sigs.k8s.io/yaml is not local module so it should not be grouped with local ones.